+seq/Read.m

```diff
@@ -1,23 +1,24 @@
 %{
-# Sequence reads from each lane
+# 
 -> seq.Lane
-read_id:  int unsigned   # read number within the lane 
------
-read_seq: varchar(100)  # sequence of actual read
-read_qc: varchar (100) # quality control measure for the read sequence
-index_seq: varchar (50) # sequence of the index
-index_qc: varchar(100) # quality control measure for the index sequence
-
+read_id                     : char(16)                      # machine-assigned id
```

Comment:
i don't think there is useful information there and it should be ok to use a number